package.json

```diff
@@ -48,7 +47,7 @@
     "@commitlint/cli": "16.2.3",
     "@commitlint/config-conventional": "16.2.1",
     "@rollup/plugin-babel": "5.3.1",
-    "@rollup/plugin-node-resolve": "13.0.5",
+    "@rollup/plugin-node-resolve": "13.1.3",
```

Comment:
Let's keep 13.0.5 for consistency with Locker. We pin it there for BEST compat (until BEST updates their rollup).